models/src/main/scala/it/pagopa/interop/authorizationmanagement/model/client/PersistentClient.scala

```diff
@@ -9,5 +10,6 @@ final case class PersistentClient(
   purposes: Seq[PersistentClientStatesChain],
   description: Option[String],
   relationships: Set[UUID],
-  kind: PersistentClientKind
+  kind: PersistentClientKind,
+  createdAt: Option[OffsetDateTime]
```

Comment:
Hmm, you're right, it should be mandatory. We need to plan a procedure to populate this field when this PR will get merged

Reply:
In one of the other services (I don't remember which one) we hardcoded the creation timestamp to a fixed date in the deserializer. If it is a viable solution we could to the same here

Reply:
> In one of the other services (I don't remember which one) we hardcoded the creation timestamp to a fixed date in the deserializer. If it is a viable solution we could to the same here

Is it [this one](https://github.com/pagopa/interop-be-party-management/blob/0ab320225f669e7160d17f667b4fa4a1bb4e56f4/src/main/scala/it/pagopa/interop/partymanagement/model/persistence/serializer/v1/package.scala#L101)?

If we want to hardcode the timestamp, should we do it in the process instead of the management like we are doing with the other dates?

Reply:
I think it was in the catalog management.

Fixing the timestamp in the process would be nice, but in that case all existing data would still have the empty field, which force every service that use the read model to apply the default value